cmd/agent/app/httpserver/manager.go

```diff
@@ -18,13 +18,17 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package sampling
+package httpserver
 
 import (
+	"github.com/uber/jaeger/thrift-gen/baggage"
 	"github.com/uber/jaeger/thrift-gen/sampling"
 )
 
-// Manager decides which sampling strategy a given service should be using.
+// Manager decides:
+// 1) which sampling strategy a given service should be using
+// 2) which baggage restrictions a given service should be using.
 type Manager interface {
```

Comment:
we should re-thing the name Manager. Some random options
  * ClientConfigManager
  * ClientController
?